sdk/core/src/main/java/com/klaviyo/coresdk/utils/ConfigFileUtils.kt

```diff
@@ -0,0 +1,56 @@
+package com.klaviyo.coresdk.utils
+
+import com.klaviyo.coresdk.KlaviyoConfig
+import java.io.File
+import java.util.UUID
+
+internal object ConfigFileUtils {
+    private const val CONFIG_FILE_NAME = "klaviyo-sdk-config"
+
+    private fun openConfigFile(): File {
+        return File("${KlaviyoConfig.applicationContext.filesDir}/$CONFIG_FILE_NAME")
+    }
+
+    fun readOrCreateUUID(): String {
+        val file = openConfigFile()
+
+        var uuid = findConfigValue(file, ConfigKeys.UUID_KEY)
+
+        if (uuid.isEmpty()) {
+            uuid = UUID.randomUUID().toString()
+            writeConfigValue(file, ConfigKeys.UUID_KEY, uuid)
+        }
+
+        return uuid
+    }
+
+    fun readValue(key: String): String {
+        return findConfigValue(openConfigFile(), key)
+    }
+
+    fun findConfigValue(file: File, key: String): String {
+        var value = ""
+
+        if (!file.exists()) {
+            return value
+        }
+
+        file.useLines { lines ->
+            lines.forEach { line ->
+                if (line.startsWith("$key:")) {
+                    value = line.substring("$key:".length)
+
+                }
+            }
+        }
+        return value
+    }
+
+    fun writeConfigValue(key: String, value: String) {
+        writeConfigValue(openConfigFile(), key, value)
+    }
+
+    fun writeConfigValue(file: File, key: String, value: String) {
+        file.appendText("$key:$value\n")
```

Comment:
I was using `writeText` here before which is incorrect, I want to use `appendText` so that the file can contain multiple values (UUIDs and push tokens right now). Whenever either of these need to be generated, the file will be deleted anyway since they are only generated on initial install